### DISCUSSION TITLE
The place of Duration Formatter

### DISCUSSION THREAD

**Discussion by younies** [unicode-org/icu4x#4956] [Duration Formatter]
For the Duration Formatter, shall we locate it under the `experimental/dimension` or add a new section `experimental/duration` 

Discussion:
@sffc @DesmondWillowbrook @younies

**Comment by younies** [comment]
For start, `experimental/duration` and we can move it after the decision.

**Reply by sffc** [reply]
Yes, please start in `experimental/duration`.
